### ISSUE TITLE
No visual indicator to separate different days' notifications

### ISSUE THREAD

**Issue by ScenarioKG** [wordpress-mobile/WordPress-Android#21191]
### Expected behavior
Notifications should clearly display separation between different days for better readability.

### Actual behavior
Notifications from different days appear without clear separators, potentially confusing users about the timeline of events.

### Steps to reproduce the behavior
1. Open 'Notifications'
2. Navigate to notifications older than a few days.

##### Tested on [device], Android [version], JPAndroid / WPAndroid [version]
Galaxy S21 5G, Android 12, Jetpack 25.2.1 (1444)/WordPress 25.2.1 (1444)

![SCREENSHOT_1](https://github.com/user-attachments/assets/e3ae66b9-68d2-4150-b87d-e4ea94a4ddf9)

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by aaishwarymishra** [comment]
Hi , can you guide me a little like which file handle's this , I am new

**Comment by nbradbury** [comment]
@BLANKatGITHUB I _think_ `ui.notifications.adapters.NotesAdapter` is what you're looking for.

**Comment by Anubhav8176** [comment]
Hi! Anyone working on it Or not? If not, I want to work on the issue as I think I got the issue pretty well.

**Comment by nbradbury** [comment]
@Anubhav8176 Apologies for the delay - I took some time off recently. Yes, please feel free to work on this. Thank you!

**Comment by Anubhav8176** [comment]
@nbradbury Can you please guide me to which file/folder should I look for the issue? As I am very new to the open source community

**Comment by nbradbury** [comment]
> Can you please guide me to which file/folder should I look for the issue?

As mentioned above, I _think_ `ui.notifications.adapters.NotesAdapter` is what you're looking for.

**Comment by Anubhav8176** [comment]
@nbradbury I made few changes in the code in buildFilteredNotesList function and now i want to test that the changes are reflected or not in the app. Where I can check the notification options in the app. I have tried running app on emulator but couldn't get to the notification screen. Can you help me with this?

**Comment by nbradbury** [comment]
@Anubhav8176 Make sure you're using a Jetpack build variant rather than a WordPress one, since notifications are only available in the Jetpack app.

![Image](https://github.com/user-attachments/assets/2a540348-819d-4720-a114-d9a058b46b59)

**Comment by Anubhav8176** [comment]
@nbradbury  
Do I need to install jetpack to see the notification, and even when I try to click on install jetpack it asks the email and password when i enter it just shows unknown client_id. I also tried to go through the documentation and couldn't find any satisfying answer to this problem.

![Image](https://github.com/user-attachments/assets/2fc2535a-df47-410d-a1bd-71172041b716)

**Comment by nbradbury** [comment]
@Anubhav8176 Yes, you do need to have the Jetpack app installed to see notifications, and your site must also be using the Jetpack plugin. Are you using the Jetpack plugin?

**Comment by Anubhav8176** [comment]
Yes I have created a test site using jurassic.ninja (mentioned in README.md) and when i see the installed plugins it mentioned jetpack is installed.

![Image](https://github.com/user-attachments/assets/54166d2a-6759-49e6-9722-3975ce43ef35)

**Comment by nbradbury** [comment]
@Anubhav8176 Okay, so the problem, then, is that you're seeing "unknown client_id" when attempting to login? I'll check with our team about this.

**Comment by nbradbury** [comment]
@Anubhav8176 I checked with the team and unfortunately the problem is that our documentation is incomplete, in that it's missing instructions for Jetpack builds. We'll resolve this but it's unlikely to happen soon. For that reason, if you still wish to contribute, I recommend choosing something that's not Jetpack-specific.
